library/coqlib.ml

```diff
@@ -227,8 +227,7 @@ type coq_eq_data = {
 
 (* Leibniz equality on Type *)
 
-let build_eqdata_gen lib str =
-  let _ = check_required_library lib in {
```

Comment:
This change was missing in the main coqlib PR right?

Reply:
I guess it is a "bug" that prevented the rest of this PR to be tested. The whole point of the new coqlib is to be independent, the the removed code imposes a dependency on some library even if entry was rebound.